<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CleanerClass</class>
 <widget class="QMainWindow" name="CleanerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>垃圾清理V1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Cleaner.qrc">
    <normaloff>:/Cleaner/res/垃圾清理图标.png</normaloff>:/Cleaner/res/垃圾清理图标.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(209, 209, 209);</string>
        </property>
        <property name="title">
         <string>文件扫描结果</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout" stretch="1">
           <item>
            <widget class="QTreeView" name="treeView"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(209, 209, 209);</string>
        </property>
        <property name="title">
         <string>操作</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1,1,1,1">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="start">
             <property name="text">
              <string>开始扫描</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stop">
             <property name="text">
              <string>停止扫描</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>用时</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="time"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>扫描文件数目</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="fCountN"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="select_all">
             <property name="text">
              <string>全选</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="unselect">
             <property name="text">
              <string>全否</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clear_all">
             <property name="text">
              <string>删除选中文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="unfold_all">
             <property name="text">
              <string>全部展开</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fold_all">
             <property name="text">
              <string>全部折叠</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Cleaner.qrc"/>
 </resources>
 <connections/>
</ui>
